scripts/start-finalExec

```diff
@@ -96,11 +96,12 @@ patchLog4jConfig() {
 }
 
 # Patch Log4j remote code execution vulnerability
-if [[ ${TYPE^^} != PAPER ]] && versionLessThan 1.12; then
+# NOTE: Paper, bukkit, and spigot are normalized to SPIGOT
+if [[ ${TYPE^^} != SPIGOT ]] && versionLessThan 1.12; then
   patchLog4jConfig log4j2_17-111.xml https://launcher.mojang.com/v1/objects/dd2b723346a8dcd48e7f4d245f6bf09e98db9696/log4j2_17-111.xml
-elif [[ ${TYPE^^} != PAPER ]] && versionLessThan 1.17; then
+elif [[ ${TYPE^^} != SPIGOT ]] && versionLessThan 1.17; then
   patchLog4jConfig log4j2_112-116.xml https://launcher.mojang.com/v1/objects/02937d122c86ce73319ef9975b58896fc1b491d1/log4j2_112-116.xml
-else
+elif versionLessThan 1.18; then
```

Comment:
I do believe 1.18 requires the fix. Only 1.18.1 is not affected.
![image](https://user-images.githubusercontent.com/19885942/145608816-b365f054-39fd-49e5-9d16-4a79c7b2fa0a.png)

Reply:
Yep, caught that midflight while reading your previous comment.